operational_analysis/methods/plant_analysis.py

```diff
@@ -545,20 +545,16 @@ def trim_monthly_df(self):
     @logged_method_call
     def calculate_long_term_losses(self):
         """
-        This function calculates long-term availability and curtailment losses based on the reported data,
-        filtering for those data that are deemed representative of average plant performance
+        This function calculates long-term availability and curtailment losses based on the reported data grouped by the time resolution,
+        filtering for those data that are deemed representative of average plant performance.
         
         Args:
             (None)
         
         Returns:
-            (tuple):
-                :obj:`float`: long-term annual availability loss expressed as fraction
-                :obj:`float`: long-term annual curtailment loss expressed as fraction
+            (None)
         """
-        df = self._aggregate.df
-        
-        days_year_lt = 365.25 # Number of days per long-term year (accounting for leap year every 4 years)
+        df = self._aggregate.df.rename_axis(None)
```

Comment:
Just a minor comment: why do we need the .rename_axis(None) here? Maybe let's add a little comment to clarify that?

Reply:
this was needed to combine with the gross_lt Pandas Series in sample_long_term_losses. Multiplying them wouldn't work unless they had the same axis names (and gross_lt didn't have a name). But I decided to change it to keep the axis name 'time' and instead renamed the gross_lt axis to 'time' in sample_long_term_losses (with a comment there too)